python/cuml/test/test_fit_function.py

```diff
@@ -60,6 +61,8 @@ def test_fit_function(dataset, model_name):
         # to register the number of arguments manually if `fit` is decorated
         pos_args_spec = {
             "ARIMA": 1,
+            'DBSCAN': 2,
+            'NearestNeighbors': 2
```

Comment:
Why did you have to add these? If I delete DBSCAN it still works just fine.

Reply:
I think a former iteration of the decorator made these not work, will check again